From c677303f423d433a16f74c7b78ea015bffcdb347 Mon Sep 17 00:00:00 2001
From: Imran Rashid <irashid@cloudera.com>
Date: Thu, 11 May 2017 16:00:12 -0500
Subject: [PATCH 508/517] SPARK-12297.  Fix Failing Unit test for Parquet
 Timestamps.

The last change introduced a new entry to hadoop conf which I forgot to
update in this unit test in the last patch.

Cloudera ID: CDH-53738
---
 .../parquet/ParquetTimestampSuite.scala            |    1 +
 1 file changed, 1 insertion(+)

diff --git a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTimestampSuite.scala b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTimestampSuite.scala
index c2ef481..e97b249 100644
--- a/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTimestampSuite.scala
+++ b/sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetTimestampSuite.scala
@@ -53,6 +53,7 @@ class ParquetTimestampSuite extends SparkFunSuite with MockitoSugar with SharedS
       val catalystSchema = StructType(Array(StructField("ts", TimestampType, true)))
       CatalystWriteSupport.setSchema(catalystSchema, conf)
       conf.set(ParquetFileFormat.PARQUET_TIMEZONE_TABLE_PROPERTY, "America/Chicago")
+      conf.set(ParquetFileFormat.FILE_CREATOR, "parquet-mr")
       conf.set(SQLConf.PARQUET_WRITE_LEGACY_FORMAT.key, "true")
       conf.set(SQLConf.PARQUET_BINARY_AS_STRING.key, "false")
       conf.set(SQLConf.PARQUET_INT96_AS_TIMESTAMP.key, "true")
-- 
1.7.9.5

